transformer-intro/run-me.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-MARIAN=../../build
+if [ -z ${MARIAN+x} ]; then MARIAN=../../build; fi
```

Comment:
I'm not sure what you wanted to achieve with the if statement here. Can you explain? If you tried to use MARIAN envvar if it's set, wouldn't something like that below work? Thanks
```
MARIAN=${MARIAN:-../../build}
```

Reply:
I wanted to achieve exactly what you are doing, but I didn't know about the `:-` operator. Thanks for letting me know about it; it's more concise. I'll update the PR accordingly.